en/docs/learn/enterprise-integration-patterns/messaging-systems/pipes-and-filters.md

```diff
@@ -1,119 +1,130 @@
+This page explains how you can implement a sample scenario of Pipes and Filters EIP using WSO2 Micro Integrator.
+
+## Introduction to Pipes and Filters EIP
+
+The Pipes and Filters EIP breaks down a large task into smaller subsets of independent steps chained together. This is useful when a sequence of processing steps is required to perform a single event in an integration scenario. The main use case of this EIP is to maintain independence and flexibility between each processing step.
 
 !!! info
     For more information, see the [Pipes and Filters](https://www.enterpriseintegrationpatterns.com/patterns/messaging/PipesAndFilters.html) documentation.
 
 ## Sample scenario
 
-The example scenario depicts how a stock quote request is sent from a client to a service (Stock Quote Service). The request is first received by the ESB profile, which transmits the request through the following filters:
+The example scenario depicts how a stock quote request is sent from a client to a service (Stock Quote Service). The request is first received by the WSO2 MI, which transmits the request through the following filters:
 
 * Check Username filter: to verify the username.
 * Check User ID filter: to verify the user ID.
 
 !!! note
-    Filter Mediators are used in the ESB, to verify the validity of the message (checking the username and user ID).
+    Filter Mediators are used in the service to verify the validity of the message (checking the username and user ID).
 
-If the message meets the criteria of the first filter, it will be passed to the second filter. Once the request successfully passes through the second filter (i.e., the filtering criteria is fulfilled), the ESB sends the stock quote request to the back-end service (Stock Quote Service) for processing.
+If the message meets the criteria of the first filter, it passes to the second filter. Once it passes the second filter, the service sends the stock quote request to the back-end service (Stock Quote Service) for processing.
 
 ![Pipes and filters]({{base_path}}/assets/img/learn/enterprise-integration-patterns/messaging-systems/pipes-and-filters.png)
 
 ## Synapse configurations of the artifacts
 
 !!! note
-    When you unzip the ZIP file you download below in Step 6 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/synapse-config` directory. For more information about these artifacts, go to WSO2 EI Documentation.
+    When you unzip the ZIP file you downloaded below in step 7 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/wso2mi/artifacts` directory. For more information about these artifacts, go to WSO2 MI Documentation.
```

Comment:
'For more information about these artifacts, go to WSO2 MI Documentation.' Let's point to synapse artifacts page, as we are already in MI docs